.changeset/cold-news-glow.md

```diff
@@ -0,0 +1,14 @@
+---
+"@localyze-pluto/components": major
+---
+
+[ContentCard]
+
+> Breaking change: `iconUrl` prop is now `icon` and accepts a ReactNode
+
+Minor changes:
+
+- Set Icon wrapper border radius to 8px (shows now like a square)
+- Added new `topRightElement` prop to allow add element in the top right corner of the card
+- Added new `lowOpacity` prop to set the opacity of the card content without include the `topRightElement`
```

Comment:
nit: why not pass `opacity` (as a number) instead of lowOpacity ? It feels like we are not adding a prop but rather using one of the default css props

Reply:
I’m not sure I understand what you mean. You mean the prop name? Do you think renaming it to `hasLowOpacity` would be better? 
I didn’t add it as a number since it’s a specific opacity value. In our case at Localyze, we want to apply a lower opacity (0.6) when the partner is disabled. I avoided using disabled as a prop because the card remains interactive.

Reply:
I meant a number, yes. I know it's a specific number but right now it feels like we have so many specific props for this card. Anyway, not a blocker